app/src/main/java/com/example/cachua/musicdemo21/data/model/MusicModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.cachua.musicdemo21.data.model
+
+import android.net.Uri
+
+data class MusicModel(var id:Int,
```

Comment:
```suggestion
data class MusicModel(val id:Int,
```
Nên dùng val nếu không muốn properties thay đổi

Reply:
Anh nên viết như này nhé, chú ý một đặt tên field là phải là 1 danh từ, clear. 
Mỗi file cần để 1 dòng trống ở cuối
```suggestion
data class MusicModel(
    val id: Int,
    val artist: String,
    val title: String,
    val data: String,
    val name: String,
    val duration: Int,
    val albumId: Long,
    val uri: Uri
)
```